src/internal/operators/mergeMap.ts

```diff
@@ -1,9 +1,9 @@
-import { ObservableInput, OperatorFunction, ObservedValueOf } from '../types';
-import { map } from './map';
-import { innerFrom } from '../observable/from';
-import { operate } from '../util/lift';
-import { mergeInternals } from './mergeInternals';
-import { isFunction } from '../util/isFunction';
+import { ObservableInput, OperatorFunction, ObservedValueOf } from '../types.js';
```

Comment:
for type only imports it's still need to ref to .js while it's empty? (https://unpkg.com/browse/rxjs@7.0.0-beta.9/dist/cjs/internal/types.js)